applications/dashboard/controllers/Api/TokensApiController.php

```diff
@@ -0,0 +1,316 @@
+<?php
+/**
+ * @copyright 2009-2017 Vanilla Forums Inc.
+ * @license GPLv2
+ */
+
+use Garden\Schema\Schema;
+use Garden\Web\Exception\ClientException;
+use Garden\Web\Exception\NotFoundException;
+
+/**
+ * API Controller for the `/tokens` resource.
+ */
+class TokensApiController extends AbstractApiController {
+
+    /** Default expiry for issued tokens. */
+    const DEFAULT_EXPIRY = '10 years';
+
+    /** The maximum number of tokens in a response. */
+    const RESPONSE_LIMIT = 200;
```

Comment:
If we limit a response we need pagination.

Not having pagination make the possibility of having an issue not being able to get limit + 1 result even if the limit is stupidly high.
Not having ensure that we will never have that problem.
If we want to impose a limit for scaling reason we should do so on the create endpoint, not the get.